### ISSUE TITLE
"pm2 gc" command throws "TypeError: pm2.forceGc is not a function"

### ISSUE THREAD

**Issue by Gawdl3y** [Unitech/pm2#2392]
#### Expected behaviour
`pm2 gc` to trigger garbage collection

#### Actual behaviour
`pm2 gc` throws an error

```
/usr/lib/node_modules/pm2/bin/pm2:765
    pm2.forceGc();
        ^

TypeError: pm2.forceGc is not a function
    at Command.<anonymous> (/usr/lib/node_modules/pm2/bin/pm2:765:9)
    at Command.listener (/usr/lib/node_modules/pm2/node_modules/commander/index.js:301:8)
    at emitTwo (events.js:106:13)
    at Command.emit (events.js:191:7)
    at Command.parseArgs (/usr/lib/node_modules/pm2/node_modules/commander/index.js:615:12)
    at Command.parse (/usr/lib/node_modules/pm2/node_modules/commander/index.js:458:21)
    at beginCommandProcessing (/usr/lib/node_modules/pm2/bin/pm2:113:13)
    at /usr/lib/node_modules/pm2/bin/pm2:176:7
    at /usr/lib/node_modules/pm2/lib/API.js:136:14
    at /usr/lib/node_modules/pm2/lib/Client.js:52:16
```

#### Steps to reproduce
Use `pm2 gc`

#### Software versions used
```
OS         : Ubuntu Server 16.04.1 LTS
node.js    : 6.6.0
PM2        : 2.0.12
```

**Comment by Unitech** [comment]
This command is not for public use, will remove it

**Comment by fredericosilva** [comment]
@Unitech it is still available on 2.7.2 is it safe to use?

**Comment by mrfelton** [comment]
Still available in 2.10.1 and still throws the same error.

**Comment by wallet77** [comment]
I will remove it.

**Comment by TheEvilSocks** [comment]
Still available in 2.10.2, and same error.
```
/usr/lib/node_modules/pm2/bin/pm2:900
    pm2.forceGc();
        ^

TypeError: pm2.forceGc is not a function
    at Command.<anonymous> (/usr/lib/node_modules/pm2/bin/pm2:900:9)
    at Command.listener (/usr/lib/node_modules/pm2/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:106:13)
    at Command.emit (events.js:194:7)
    at Command.parseArgs (/usr/lib/node_modules/pm2/node_modules/commander/index.js:653:12)
    at Command.parse (/usr/lib/node_modules/pm2/node_modules/commander/index.js:475:21)
    at beginCommandProcessing (/usr/lib/node_modules/pm2/bin/pm2:145:13)
    at /usr/lib/node_modules/pm2/bin/pm2:219:7
    at /usr/lib/node_modules/pm2/lib/API.js:169:14
    at /usr/lib/node_modules/pm2/lib/Client.js:54:16

```

**Comment by adamjeffery-dev** [comment]
Still available in 2.10.4

/usr/lib/node_modules/pm2/bin/pm2:900
    pm2.forceGc();
        ^

TypeError: pm2.forceGc is not a function
    at Command.<anonymous> (/usr/lib/node_modules/pm2/bin/pm2:900:9)
    at Command.listener (/usr/lib/node_modules/pm2/node_modules/commander/index.js:315:8)
    at emitTwo (events.js:126:13)
    at Command.emit (events.js:214:7)
    at Command.parseArgs (/usr/lib/node_modules/pm2/node_modules/commander/index.js:653:12)
    at Command.parse (/usr/lib/node_modules/pm2/node_modules/commander/index.js:475:21)
    at beginCommandProcessing (/usr/lib/node_modules/pm2/bin/pm2:145:13)
    at /usr/lib/node_modules/pm2/bin/pm2:219:7
    at /usr/lib/node_modules/pm2/lib/API.js:169:14
    at /usr/lib/node_modules/pm2/lib/Client.js:54:16

**Comment by wallet77** [comment]
Guys, it's fixed but still in development branch.
We will publish it with pm2 3.0.0
